Beatmap Spotlights Spring 2022 conclusion newspost (#7716)
ppy/osu-wiki: news/2022-07-28-beatmap-spotlights-spring-2022-conclusion.md

---
layout: post
title: "Beatmap Spotlights Spring 2022 Conclusion & Players' Pick Voting"
date: 2022-07-28 16:00:00 +0000
---

> The Spring 2022 Season of the Beatmap Spotlights has come to an end⁠ — come check out the top 50 for each game mode, and vote on your favourite beatmaps from the season! Join us on Summer 2022 Season launch livestream as well.

Reviewer: ```suggestion
The Spring 2022 season of the Beatmap Spotlights has come to an end⁠ — come check out the top 50 for each game mode, and vote on your favourite beatmaps from the season! Join us on the Summer 2022 season launch livestream as well.
```

![](/wiki/shared/news/banners/beatmap-spotlights.jpg)

> Continuing the trend from past seasons, Spring 2022 playlists gathered nearly **2900** players either competing for fun or jumping on a grindwagon to gain this *Rhythm Incarnate* rank. Since the last season's Players' Pick turned out gripping for both players and mappers, we will be running it again this season, read on below for that and even more exciting new stuff!

Reviewer: ```suggestion
Continuing the trend from past seasons, Spring 2022 playlists gathered nearly **2900** players either competing for fun or jumping on a grindwagon to gain this *Rhythm Incarnate* rank. Since the last season's Players' Pick turned out gripping for both players and mappers, we will be running it again this season — read on below for that and even more exciting new stuff!
```

## Livestream

> Livestream? Yes, you've heard right! We will be streaming Summer 2022 Season start over our [twitch channel](https://www.twitch.tv/osu_spotlights) at **August 7th, 16:00 UTC+0**.

Reviewer: ```suggestion
Livestream? Yes, you've heard right! We will be streaming the Summer 2022 Season start over at our [Twitch channel](https://www.twitch.tv/osu_spotlights) at **August 7th, 16:00 UTC+0**.
```

> Livestream? Yes, you've heard right! We will be streaming Summer 2022 Season start over our [twitch channel](https://www.twitch.tv/osu_spotlights) at **August 7th, 16:00 UTC+0**.

Reviewer 2: ```suggestion
Livestream? Yes, you've heard it right! We will be streaming Summer 2022 Season start on our [Twitch channel](https://www.twitch.tv/osu_spotlights) at **August 7th, 16:00 UTC+0**.
```

> The stream will last for around 2 hours, and will serve as an introduction to the new season including things like season gimmick and playlist A picks reveal, commented by some of our [Curators](/wiki/Beatmap_Spotlights#curators). 

Reviewer: ```suggestion
The stream will last for around 2 hours, and will serve as an introduction to the new season including things like the season gimmick and revealing the playlist A picks, commented by some of our [curators](/wiki/Beatmap_Spotlights#curators). 
```

This will be the very first time of it taking a place, so we hope you're excited and count on your presence!

> ## Beatmap of the Season — Players' Pick

Reviewer: :)

```suggestion
## Beatmap of the Season - Players' Pick
```

> As stated before, the *Beatmap of the Season* - Players' Pick will be a regular thing from now on, combined with [Spotlights Curators](/wiki/Beatmap_Spotlights#curators) one, similar to last, Winter 2022 Season voting.

Reviewer: ```suggestion
As stated before, the *Beatmap of the Season - Players' Pick* will be a regular thing from now on, combined with the [Spotlights curators'](/wiki/Beatmap_Spotlights#curators) one, similar to last, Winter 2022 Season voting.
```

> As stated before, the *Beatmap of the Season* - Players' Pick will be a regular thing from now on, combined with [Spotlights Curators](/wiki/Beatmap_Spotlights#curators) one, similar to last, Winter 2022 Season voting.

Reviewer 2: ```suggestion
As stated before, the *Beatmap of the Season - Players' Pick* will be a regular thing from now on, combined with [Spotlights Curators](/wiki/Beatmap_Spotlights#curators) one, similar to last, Winter 2022 Season voting.
```

shouldnt the italic be like that?

> As stated before, the *Beatmap of the Season - Players' Pick* will be a regular thing from now on, combined with the [Spotlights curators'](/wiki/Beatmap_Spotlights#curators) picks, similarly to how it ran last time in the Winter 2022 Season voting.

Reviewer: ```suggestion
As stated before, the *Beatmap of the Season - Players' Pick* will be a regular thing from now on, combined with the [Spotlights curators'](/wiki/Beatmap_Spotlights#curators) picks, similarly to how it ran last time in the Winter 2022 season voting.
```

The results of both the curator and player picks will be announced at the launch of the Summer 2022 season of the Beatmap Spotlights, and the creators of winning entries will be awarded with **3 months of osu!supporter** each. If you'd like to make your opinions known on the best maps of the season, this is the place to do it!

Check out each respective game mode's contest and vote today:

- [osu!](https://osu.ppy.sh/community/contests/149)
- [osu!taiko](https://osu.ppy.sh/community/contests/150)
- [osu!catch](https://osu.ppy.sh/community/contests/151)
- [osu!mania](https://osu.ppy.sh/community/contests/152)

> The voting will remain open for **7 days** from the date of this news post, so be sure to vote in time!

Reviewer 2: ```suggestion
The voting will remain open for **7 days** from the date of this news post, so make sure to vote in time!
```

> ## Beatmap Spotlights final rankings

Reviewer: ```suggestion
## Beatmap Spotlights Spring 2022 final rankings
```

> [Click here for the full leaderboards of every game mode!](https://docs.google.com/spreadsheets/d/e/2PACX-1vSJ96g-8FrD_rtJ3bx3TDfHw1RLg5jvasQKsf9V-c_nO1y_E08JduIsWZbqiDJ7bxyTy4CG3VYSkpcc/pubhtml#)

Reviewer 2: ```suggestion
You can check the full leaderboards of every game mode [at this spreadsheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vSJ96g-8FrD_rtJ3bx3TDfHw1RLg5jvasQKsf9V-c_nO1y_E08JduIsWZbqiDJ7bxyTy4CG3VYSkpcc/pubhtml#).
```

Reviewer 3: i think this one is more accessible as is

| #49 | [LewusekX44](https://osu.ppy.sh/users/9997359) | ![][spring-2022-osu!-diamond-2] | [Horiiizon](https://osu.ppy.sh/users/8071438) | ![][spring-2022-taiko-gold-2] | [a16837295416](https://osu.ppy.sh/users/16680343) | ![][spring-2022-ctb-gold-2] | [Teaful](https://osu.ppy.sh/users/16972265) | ![][spring-2022-mania-gold-3] |
| #50 | [Alvaroping1](https://osu.ppy.sh/users/15289150) | ![][spring-2022-osu!-diamond-2] | [Boaz](https://osu.ppy.sh/users/13302996) | ![][spring-2022-taiko-gold-2] | [gorkem_xd](https://osu.ppy.sh/users/27415754) | ![][spring-2022-ctb-gold-2] | [stuffisherenow](https://osu.ppy.sh/users/14177626) | ![][spring-2022-mania-gold-3] |

---

Join us over at the [osu! community Discord server](https://discord.gg/0Vxo9AsejDkGlk3H) and assign yourself a role to keep up with Spotlights changes and announcements, or visit the [osu!dev Discord server](https://discord.gg/ppy) to get involved and help us push this project even further beyond!

You can find out more about the Beatmap Spotlights [on the osu! wiki](/wiki/Beatmap_Spotlights).

> See you soon along with the Summer Season 2022 launch!

Reviewer: ```suggestion
See you soon along with the Summer 2022 season launch!
```
